syncer/syncer_test.go

```diff
@@ -791,115 +783,21 @@ func (s *testSyncerSuite) TestColumnMapping(c *C) {
 	}
 }
 
-func (s *testSyncerSuite) TestTimezone(c *C) {
-	s.cfg.BWList = &filter.Rules{
-		DoDBs:     []string{"~^tztest_.*"},
-		IgnoreDBs: []string{"stest", "~^foo.*"},
-	}
-
-	createSQLs := []string{
-		"create database if not exists tztest_1",
-		"create table if not exists tztest_1.t_1(id int, a timestamp)",
-	}
-
-	testCases := []struct {
-		sqls     []string
-		timezone string
-	}{
-		{
-			[]string{
-				"insert into tztest_1.t_1(id, a) values (1, '1990-04-15 01:30:12')",
-				"insert into tztest_1.t_1(id, a) values (2, '1990-04-15 02:30:12')",
-				"insert into tztest_1.t_1(id, a) values (3, '1990-04-15 03:30:12')",
-			},
-			"Asia/Shanghai",
-		},
-		{
-			[]string{
-				"insert into tztest_1.t_1(id, a) values (4, '1990-04-15 01:30:12')",
-				"insert into tztest_1.t_1(id, a) values (5, '1990-04-15 02:30:12')",
-				"insert into tztest_1.t_1(id, a) values (6, '1990-04-15 03:30:12')",
-			},
-			"America/Phoenix",
-		},
-	}
-	queryTs := "select unix_timestamp(a) from `tztest_1`.`t_1` where id = ?"
-
-	dropSQLs := []string{
-		"drop table tztest_1.t_1",
-		"drop database tztest_1",
-	}
-
-	for _, sql := range createSQLs {
-		s.db.Exec(sql)
+func (s *testSyncerSuite) TestGeneratedColumn(c *C) {
+	// TODO Currently mock eventGenerator don't support generate json,varchar field event, so use real mysql binlog event here
```

Comment:
```suggestion
	// TODO Currently mock eventGenerator don't support generate json, varchar field event, so use real mysql binlog event here
```

Reply:
if still use real mysql, I think we can keep the function `resetBinlogSyncer`

Reply:
keep the function but not use it?